### PR TITLE
fix: documentation for 1.2.2 release

### PR DESCRIPTION
**Pull request by fnaranjo-vmw**
Which other branches should this be merged with (if any)? none

### CODE REVIEW

File: reference/gcp-postgresql.html.md.erb [hunk 1]
```diff
@@ -276,42 +276,6 @@ following reasons:
 
 Currently, the Google PostgreSQL service does not provide a mechanism for TLS certificate rotation.
 
-## <a id="binding-known-issue"></a> Binding Process Known Issue
```
Comment (pivotal-marcela-campo) [inline reference/gcp-postgresql.html.md.erb:279]: This issue is linked from the 1.2.1 release notes. I would not remove it, but add a note at the top saying this has been fixed in version 1.2.2 instead. We can remove it from main if it is there for sure. But not from 1.2, as the docs are shared across all patch versions.

File: release-notes.html.md.erb [hunk 2]
```diff
@@ -7,6 +7,24 @@ owner: Cloud Service Broker
...
+
+### Resolved Issues
+
+This release has the following fixes:
+
+<%= partial '/csb/logo-resolved' %>
+
+#### Google Cloud PostgreSQL
+
+- **Fix permission issues on binding creation**
```
Comment (pivotal-marcela-campo) [inline release-notes.html.md.erb:20]: Usually, this refers to a known issue, so we need to make sure the wording is similar to the known issue documented previously. From the release notes for 1.2.1 further below:

```
- **Object access issues in Google PostgreSQL:**
  If an app creates an object, such as a table, with permissions that do not allow other users to see
  that object, subsequent bindings might fail, and other apps might fail to read the objects.
  For more information, see
  [Binding Process Known Issue](reference/gcp-postgresql.html.md.erb#binding-known-issue).
```

**Review by pivotal-marcela-campo — CHANGES_REQUESTED** [review]

File: release-notes.html.md.erb (hunk 2 above)
Comment (fnaranjo-vmw) [reply release-notes.html.md.erb]: Thanks. Pushed a commit with suggested changes:
https://github.com/pivotal-cf/docs-csb-gcp/pull/110/commits/64db627f320f88fae964087176915c489b66efb1

File: reference/gcp-postgresql.html.md.erb [hunk 3]
```diff
@@ -278,6 +278,20 @@ Currently, the Google PostgreSQL service does not provide a mechanism for TLS ce
 
 ## <a id="binding-known-issue"></a> Binding Process Known Issue
 
+<p class="note">
+<strong>Note:</strong> This issue has been fixed in version <a href="release-notes.html#1-2-2">1.2.2</a>.
+</p>
+
+## <a id="parameters"></a> Configuration Parameters
```
Comment (pivotal-marcela-campo) [inline reference/gcp-postgresql.html.md.erb:285]: I think this shouldn't be here

**Review by pivotal-marcela-campo — CHANGES_REQUESTED** [review]

**Review by pivotal-marcela-campo — APPROVED** [review]

**Pull request merged**